clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/userdata/LocalFileUserDataProvider.groovy

```diff
@@ -52,8 +54,12 @@ class LocalFileUserDataProvider implements UserDataProvider {
     for (int i = 0; i < maxRetry; i++) {
       try {
         return result.call()
-      } catch (RetrofitError re) {
-        if (re.kind == RetrofitError.Kind.NETWORK || (re.kind == RetrofitError.Kind.HTTP && retryStatus.contains(re.response.status))) {
+      } catch (SpinnakerHttpException e) {
+        if (retryStatus.contains(e.getResponse().getStatus())) {
+          Thread.sleep(retryBackoff)
```

Comment:
Oh my.

Can you do me a good one and add a `TODO(rz)` comment here to replace this with retry logic with something standard? 🤗

Reply:
Yes :hugs: